### DISCUSSION TITLE
Any plans for this project after SharePoint Add-In retirement in Microsoft 365 (April 2026)?

### DISCUSSION THREAD

**Discussion by LoneWolfeSharePoint** [microsoft-search/pnp-modern-search#3531] [General]
With the upcoming SharePoint Add-In retirement in Microsoft 365 (link below), Microsoft says that SharePoint Add-Ins will stop working for existing tenants and will be fully retired around April 2nd, 2026. Are there any existing plans to keep PnP Modern Search working? Or do you anticipate SPO adopting similar functionality by that time?

https://learn.microsoft.com/en-us/sharepoint/dev/sp-add-ins/retirement-announcement-for-add-ins

Many thanks.

**Comment by Amshaei** [comment]
I would love to join this discussion just to solidify what I think I know. My understanding is that many SharePoint apps used the "Add-In Model", which has since become outdated. Anything that lives on the store is fine for now, but I think that the main goal is to move into SPFx (SharePoint Framework). 

The SharePoint Framework seems to be where it's at. I am pretty sure that PnP Modern Search is made with SPFx? I am new to the PnP Modern Search, but I have been following Steve Corey's videos about it and believe we have access to such an amazing tool here. I would be so surprised if they stopped development now. So, I think that if they keep going with SPFx, the PnP Modern Search will become the industry standard for searching in SharePoint. I predict that there will be some copilot features down the road in addition to this? 

Again, this is just my understanding from recent research I have done. I'm posting here in hopes that someone will tell me whether I am in the right headspace about it. Please feel free to correct me :)

**Comment by wobba** [comment]
PnP modern search is not a sharepoint add-in. It is a SharePoint Framework Solution, thus not affected.
